largestNumber.js

```diff
@@ -0,0 +1,13 @@
+function largest(arr) {
+    return Math.max(...arr)
+}
+
+console.log(largest([3, 5, 1, 37, 26]))
+
+
+function largest2(arr) {
+    arr.sort();
```

Comment:
the sort should be for specifically numbers

Reply:
done)